assets/src/edit-story/app/media/utils/useFFmpeg.js

```diff
@@ -231,12 +235,23 @@ function useFFmpeg() {
    *
    * @type {boolean}
    */
-  const isTranscodingEnabled = Boolean(
+  const isUserSettingEnabled = Boolean(
     currentUser.meta?.web_stories_media_optimization
   );
 
+  /**
+   * Whether transcoding as a whole is supported.
+   *
+   * Considers user opt-in, cross-site isolation, and upload permissions.
+   *
+   * @type {boolean}
+   */
+  const isTranscodingEnabled =
```

Comment:
Friendly reminder, this will effect - https://github.com/google/web-stories-wp/pull/7528

Reply:
@spacedmonkey noted! I'll happily resolve any conflicts that should emerge.

Anything else to be done here or is this good to go otherwise?